src/h.js

```diff
@@ -28,6 +29,10 @@ export function h(nodeName, attributes) {
 		if ((child = stack.pop()) instanceof Array) {
 			for (i=child.length; i--; ) stack.push(child[i]);
 		}
+		else if (typeof ATTR_KEY === 'symbol' && child[Symbol.iterator]) {
```

Comment:
`typeof Symbol !== 'undefined'` should work with both native and polyfilled Symbol implementations.